crowdsec-docs/docs/observability/usage_metrics.md

```diff
@@ -0,0 +1,61 @@
+---
+id: usage_metrics
+title: Usage Metrics
+sidebar_position: 6
+---
+
+:::info
+Usage metrics require to run CrowdSec v1.6.3.
+
+Remediation component support is a work in progress, and we are working to add usage metrics to more of them.
+:::
+
+Logs processors and remediation components can send more detailed information about their usage to LAPI in order to provide more insights about their behavior in a single place.
+
+## Remediation Components
+
+Remediation components can send detailed information about themselves and what they are blocking.
+
+The exact metrics sent will differ based on the type of remediation component used.
+
+For example, the firewall remediation component will be able to report metrics about the amount of dropped bytes or packets and the openresty remediation component will be able to report metrics about the amount of dropped HTTP requests.
+
+The same goes for interpreting the metrics: if blocking at the firewall level, most bots/attackers will stop as soon as they see they cannot establish a connection to the target server, meaning the numbers of dropped packets or bytes should be interpreted as being relative to each other (ie, this is not the number of packets that would have been transmitted if the IP was not blocked, but more of an indication of the effectiveness of various blocking sources between each other).
```

Comment:
+ In contrast, web-based bouncers are more likely to count each individual blocked request, as attackers might not stop when receiving a 403.